options.go

```diff
@@ -0,0 +1,11 @@
+package otto
+
+// Option is an otto option
+type Option func(*Otto)
+
+// WithConsole adds a console option to the otto instance
+func WithConsole(console Console) func(o *Otto) {
```

Comment:
Don't think we need the ¬With`
```suggestion
// Console configures Otto with a specific console.
func Console(console Console) func(o *Otto) {
```

Reply:
I don't like having an interface plus function with identical name as it's confusing?

Reply:
Ahh yes, of course, ignore me.